docs/website/docs/dlt-ecosystem/destinations/bigquery.md

```diff
@@ -118,9 +118,13 @@ When staging is enabled:
 > ❗ **Bigquery cannot load JSON columns from `parquet` files**. `dlt` will fail such jobs permanently. Switch to `jsonl` to load and parse JSON properly.
 
 ## Supported column hints
+
 BigQuery supports the following [column hints](https://dlthub.com/docs/general-usage/schema#tables-and-columns):
-* `partition` - creates a partition with a day granularity on decorated column (`PARTITION BY DATE`). May be used with `datetime`, `date` data types and `bigint` and `double` if they contain valid UNIX timestamps. Only one column per table is supported and only when a new table is created.
-* `cluster` - creates a cluster column(s). Many column per table are supported and only when a new table is created.
+
+* `partition` - creates a partition with a day granularity on decorated column (`PARTITION BY DATE` or `RANGE_BUCKET`).
+  May be used with `datetime`, `date` data types and `bigint` **only if** it contains valid UNIX timestamps.
```

Comment:
could you expand just a little bit on the bigint? How does it work under the hood so the user knows what is going on.

Reply:
Maybe also add a link to bq docs: https://cloud.google.com/bigquery/docs/partitioned-tables

Reply:
I also removed the requirement that the integer must represent UNIX timestamp. This isn't strictly true since values that fall outside of the GENERATE_ARRAY method are put into their own partition.